functions/src/events/scrapeEvents.ts

```diff
@@ -154,8 +154,15 @@ class HearingScraper extends EventScraper<HearingListItem, Hearing> {
       .collection("events")
       .doc(`hearing-${String(EventId)}`)
       .get()
+
     const eventData = eventInDb.data()
+    try {
+      Hearing.check(eventData)
```

Comment:
If this throws, wouldn't it just throw again in the next `Hearing.check`?

Reply:
And just being thorough - are we sure this can successfully load a pre-existing hearing from the DB (which will not have `videoUrl`/`videoFetchedAt`/`videoAssemblyId` set at all - not even to null)?

Reply:
> If this throws, wouldn't it just throw again in the next Hearing.check?
eeps, removing that debug code.

> And just being thorough - are we sure this can successfully load a pre-existing hearing from the DB (which will not have videoUrl/videoFetchedAt/videoAssemblyId set at all - not even to null)?
good call, switching back to `optional` types so the check will pass, and conditionally adding them.